spec/factories/legal_aid_applications.rb

```diff
@@ -71,13 +71,15 @@
     end
 
     trait :with_everything do
+      after(:create) do |legal_aid_appliction|
+        legal_aid_appliction.update(outstanding_mortgage_amount: Faker::Number.decimal.to_d)
+      end
```

Comment:
Why do we need to use this `after(:create)` instead of the line you removed?

Reply:
Because one callback was resetting it under some circumstances. To be honest, it's a smell that the refactor removed.